addCarToDB.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+require_once 'vendor/autoload.php';
+
+$validation = \KoalaCars\Validators\CreateCarValidator::validateCar($_POST);
+
+if ($validation === true) {
+
+    $dbConnection = \KoalaCars\DbConnector::getDb();
+    $query = \KoalaCars\Hydrators\CreateCarHydrator::createCar($dbConnection, $_POST);
+
+    if ($query == true) {
+        header('Location: index.php');
```

Comment:
if the car is added successfully, there is no confirmation message when you get onto the index page saying it worked